### PR TITLE
Improve documentation to explain multiple convictions

### PR DESCRIPTION
**Pull request by Davidslv**
I've added references to the graphs on tests that I thought it would be beneficial,
I did not want to add them on all as it could cause potential misunderstandings.

I've also added references to the README.md so that newcomers can see it and get familiar with
the most important part of this service - the calculations.

Ticket: https://trello.com/c/kT50EDZ7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,30 @@
 This is a Rails application to enable citizens to check when their convictions are spent.
 It is based on software patterns developed for the [C100 Application][c100-application].
 
+### Documentation for calculations of convictions
+
+This service follows several rules to calculate spent dates of multiple convictions,
+we have documented all of the scenarios known to us in our tests:
+
+- [spec/services/calculators/multiples/given_scenarios_spec.rb](spec/services/calculators/multiples/given_scenarios_spec.rb)
```
Comment (Davidslv) [inline README.md:13]: I'm not sure if this needs a full link to make it work on GitHub.

Comment (zheileman) [reply README.md:13]: I think it works. You can see by going here:
https://github.com/ministryofjustice/disclosure-checker/blob/a64194c666c858dd8f483157a9d51f29a6e22681/README.md

There is an extra bullet but other than that looks perfect and links work!

**Review by zheileman — APPROVED** [review]
🎉

File: README.md (hunk 1 above)
Comment (Davidslv) [reply README.md:13]: Ah! Did not save the file when I added the last link (to the spreadsheet), fixed now

**Pull request merged**